include/aspect/material_model/rheology/drucker_prager.h

```diff
@@ -79,6 +84,17 @@ namespace aspect
           parse_parameters (const unsigned int n_fields,
                             ParameterHandler &prm);
 
+          /**
+           * Compute the steady-state angle of internal friction using the standard (static)
+           * angle of internal friction, dynamic angle of internal friction, effective strain
+           * rate, and reference strain rate.
+           */
+          double
+          compute_steady_state_friction (const double angle_internal_friction,
```

Comment:
As far as I can see this function is not really related to the drucker_prager module? All input parameters are provided from the outside. Would it make sense to move it into a separate module called `steady_state_friction_angle`? Or is the computation inside only valid for drucker prager plasticity?
In any case I would suggest renaming to `compute_steady_state_friction_angle` for clarity.